Gemfile

```diff
@@ -3,14 +3,14 @@ source "https://rubygems.org"
 
 gemspec
 
-rails_version = ENV["RAILS_VERSION"] || "4.1.4"
+rails_version = ENV["RAILS_VERSION"] || "5.0.0"
 
 gem "rails", "~> #{rails_version}"
 
 gem 'factory_girl_rails'
 gem 'sqlite3'
 gem 'rspec-rails'
-gem 'capybara', '~> 2.3.0'
```

Comment:
Yes. Older versions of Capybara use now-deprecated Rack helper method. As Rails 5 has a hard dependency on Rack 2 a version bump was required.